src/Keboola/S3Extractor/Extractor.php

```diff
@@ -2,14 +2,21 @@
 namespace Keboola\S3Extractor;
 
 use Aws\Api\DateTimeResult;
+use Aws\S3\Exception\S3Exception;
 use Aws\S3\S3Client;
 use Aws\S3\S3MultiRegionClient;
 use Monolog\Handler\NullHandler;
 use Monolog\Logger;
 use Symfony\Component\Filesystem\Filesystem;
+use Retry\RetryProxy;
+use Retry\Policy\SimpleRetryPolicy;
+use Retry\BackOff\ExponentialBackOffPolicy;
 
 class Extractor
 {
+    private const DOWNLOAD_RETRIES = 5;
+    private const DOWNLOAD_BACKOFF = 500;
```

Comment:
dalo mi to trochu zabrat rozebrat, co přesně ta konstanta určuje, co třeba `DOWNLOAD_INITIAL_INTERVAL`(`_MS`) (https://github.com/keboola/retry/blob/6a090d625f01b06ba91242832cdb575f7a5a28af/src/BackOff/ExponentialBackOffPolicy.php#L74), totéž `DOWNLOAD_MAX_ATTEMPTS` (https://github.com/keboola/retry/blob/6a090d625f01b06ba91242832cdb575f7a5a28af/src/Policy/SimpleRetryPolicy.php#L20)

Reply:
Retry samotné ano, ale ne tu konkrétní situaci. Zítra na to kouknu ještě jednou.

Reply:
^ nestačí zadat neexistující soubor, to vrací jinou exception

Reply:
^  připadaně bych mohl `SimpleRetryPolicy` neomezovat na `S3Exception`